tests/phpunit/tests/link/getEditTermLink.php

```diff
@@ -196,6 +196,25 @@ function ( $location, $term ) {
 
 		get_edit_term_link( $term, $taxonomy );
 	}
+	/**
+	 * This function checks if the return URL of `get_edit_term_link()` when called without taxonomy matches with the expected output or not.
+	 *
+	 * @ticket 61726
+	 *
+	 * @return void
```

Comment:
`@return void` is not commonly used in WordPress core.
```suggestion
```

Reply:
Fixed